content/_index.md

```diff
@@ -27,6 +27,86 @@ If you're new to KES you may want to start in one of the following pages:
 - Review set up options with the [configuration guide]({{< relref "/tutorials/configuration.md" >}}) 
 - Learn how KES works at our [Concepts page]({{< relref "concepts/_index.md" >}}).
 
+## KES Access Control
+
+KES uses mutual TLS authentication (mTLS) for performing both authentication and authorization of clients making requests to the KES server. 
+
+### Authentication
+
+Both the client and server present their x.509 certificate and corresponding private key to their peer. 
+The server only accepts the connection if the client certificate is valid and authentic:
+
+- A "valid" certificate is well-formed and current (i.e. not expired). 
+
+  Authentication with mTLS also mTLS also requires the following [Extended Key Usage](https://tools.ietf.org/html/rfc5280#section-4.2.1.12) extensions for the client and server.
+
+  - Client Authentication (`extendedKeyUsage = clientAuth`)
+  - Server Authentication (`extendedKeyUsage = serverAuth`)
+
+- An "authentic" certificate is issued by a trusted Certificate Authority (CA). 
+  Both client and server *must* include the peer certificate CA in their local system trust store.
+
+  You can start the KES server with the `kes server --auth`  option to perform mTLS with untrusted or self-signed certificates during testing or early development. 
+
+  MinIO strongly recommends only allowing trusted certificates in production environments.
+
+### Authorization
+
+KES uses Policy-Based Access Control (PBAC) to determine what operations a given client has permission to perform. 
+Each policy consists of one or more identities, where each identity corresponds to the SHA-256 hash of an x.509 certificate. 
+The server only allows the client to perform the requested operation if *all* of the following are true:
+
+- A policy on the KES server contains the client identity.
+- The policy explicitly allows the requested operation.
+
+If no such policy exists on the KES server *or* if the policy does not explicitly allow the requested operation, the KES server denies the client's request.
+
+### KES Policies
+
+KES uses policy-based access control (PBAC), where a policy describes the operations which an authenticated client may perform. 
```

Comment:
I'd drop `which` here, it seems superfluous.

Reply:
Agree